nbgrader/server_extensions/formgrader/base.py

```diff
@@ -41,7 +42,10 @@ def gradebook(self):
 
     @property
     def mathjax_url(self):
-        return self.settings['mathjax_url']
+        url = self.settings.get("mathjax_url", "")
+        if not url or url_is_absolute(url):
+            return url
+        return url_path_join(self.base_url or "/", url)
```

Comment:
This method exists in the parent `JupyterHandler` and could be removed here completely if base_url didn't include `.rstrip("/")`. 

https://github.com/jupyter-server/jupyter_server/blob/5fa7ceb3f1a5b103b9eea1231480ebddd9ae76fb/jupyter_server/base/handlers.py#L248-L261

Reply:
Do you think that `rstrip` is necessary ? Does it break something if we remove it ?

Reply:
The `rstrip` was added really early in development, so the whole server extension is built based on it. I don't think it will be difficult to change, but every usage of `{{ base_url }}/` in the templates would need to be changed to `{{ base_url }}`, and similar updates need to be made in the javascript too.